src/java/org/apache/cassandra/repair/autorepair/AutoRepair.java

```diff
@@ -167,6 +168,11 @@ public void repair(AutoRepairConfig.RepairType repairType)
             logger.debug("Auto-repair is disabled for repair type {}", repairType);
             return;
         }
+        if (!Guardrails.mixedRepairsEnabled.isEnabled(null))
```

Comment:
Could just use `enabled()` instead right?

Reply:
enabled() is if the guardrail is turned on or off, not if its predicate is true